data/json/overmap/overmap_special/aircraft_carrier.json

```
@@ -0,0 +1,170 @@
+    "id": "aircraft_carrier",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "aircraft_carrier_1a_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "aircraft_carrier_1b_north" },
+      { "point": [ 2, 0, 0 ], "overmap": "aircraft_carrier_1c_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "aircraft_carrier_2a_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "aircraft_carrier_2b_north" },
+      { "point": [ 2, 1, 0 ], "overmap": "aircraft_carrier_2c_north" },
+      { "point": [ 0, 2, 0 ], "overmap": "aircraft_carrier_3a_north" },
+      { "point": [ 1, 2, 0 ], "overmap": "aircraft_carrier_3b_north" },
+      { "point": [ 2, 2, 0 ], "overmap": "aircraft_carrier_3c_north" },
+      { "point": [ 0, 3, 0 ], "overmap": "aircraft_carrier_4a_north" },
+      { "point": [ 1, 3, 0 ], "overmap": "aircraft_carrier_4b_north" },
+      { "point": [ 2, 3, 0 ], "overmap": "aircraft_carrier_4c_north" },
+      { "point": [ 0, 4, 0 ], "overmap": "aircraft_carrier_5a_north" },
+      { "point": [ 1, 4, 0 ], "overmap": "aircraft_carrier_5b_north" },
+      { "point": [ 2, 4, 0 ], "overmap": "aircraft_carrier_5c_north" },
+      { "point": [ 0, 5, 0 ], "overmap": "aircraft_carrier_6a_north" },
+      { "point": [ 1, 5, 0 ], "overmap": "aircraft_carrier_6b_north" },
+      { "point": [ 2, 5, 0 ], "overmap": "aircraft_carrier_6c_north" },
+      { "point": [ 0, 6, 0 ], "overmap": "aircraft_carrier_7a_north" },
+      { "point": [ 1, 6, 0 ], "overmap": "aircraft_carrier_7b_north" },
+      { "point": [ 2, 6, 0 ], "overmap": "aircraft_carrier_7c_north" },
+      { "point": [ 0, 7, 0 ], "overmap": "aircraft_carrier_8a_north" },
+      { "point": [ 1, 7, 0 ], "overmap": "aircraft_carrier_8b_north" },
+      { "point": [ 2, 7, 0 ], "overmap": "aircraft_carrier_8c_north" },
+      { "point": [ 0, 8, 0 ], "overmap": "aircraft_carrier_9a_north" },
+      { "point": [ 1, 8, 0 ], "overmap": "aircraft_carrier_9b_north" },
+      { "point": [ 2, 8, 0 ], "overmap": "aircraft_carrier_9c_north" },
+      { "point": [ 0, 9, 0 ], "overmap": "aircraft_carrier_10a_north" },
+      { "point": [ 1, 9, 0 ], "overmap": "aircraft_carrier_10b_north" },
+      { "point": [ 2, 9, 0 ], "overmap": "aircraft_carrier_10c_north" },
+      { "point": [ 0, 10, 0 ], "overmap": "aircraft_carrier_11a_north" },
+      { "point": [ 1, 10, 0 ], "overmap": "aircraft_carrier_11b_north" },
+      { "point": [ 2, 10, 0 ], "overmap": "aircraft_carrier_11c_north" },
+      { "point": [ 0, 11, 0 ], "overmap": "aircraft_carrier_12a_north" },
+      { "point": [ 1, 11, 0 ], "overmap": "aircraft_carrier_12b_north" },
+      { "point": [ 2, 11, 0 ], "overmap": "aircraft_carrier_12c_north" },
+      { "point": [ 0, 12, 0 ], "overmap": "aircraft_carrier_13a_north" },
+      { "point": [ 1, 12, 0 ], "overmap": "aircraft_carrier_13b_north" },
+      { "point": [ 2, 12, 0 ], "overmap": "aircraft_carrier_13c_north" },
+      { "point": [ 0, 0, -1 ], "overmap": "aircraft_carrier_1a_-1_north" },
```

```suggestion
      { "point": [ 0, 0, -1 ], "overmap": "aircraft_carrier_1a_-1_north", "locations": [ "lake_water_cube" ] },
```

Add `"locations": [ "lake_water_cube" ]` to all underwater parts of your carrier.

---

God, i dont know what type of magic do you use, but it fixed my spawn problem
thank you